DOCUMENTATION.md

```diff
@@ -19,74 +33,74 @@ List of subsystems shared with the host system.
-PulseAudio sound server | Toggle | Allow or prohibit the application to play sounds that use PulseAudio. | `--socket=pulseaudio` and `--nosocket=pulseaudio`
-D-Bus session bus | Toggle | Allow or prohibit access to the application to the entire session bus. | `--socket=session-dbus` and `--nosocket=session-dbus`
-D-Bus system bus | Toggle | Allow or prohibit access to the application to the entire system bus. | `--socket=system-dbus` and `--nosocket=system-dbus`
-Secure Shell agent | Toggle | Allow or prohibit access to the application to SSH authentications. | `--socket=ssh-auth` and `--nosocket=ssh-auth`
-Smart cards | Toggle | Allow or prohibit access to the application to smart cards. | `--socket=pcsc` and `--nosocket=pcsc`
-Printing system | Toggle | Allow or prohibit access to the application to printing systems. | `--socket=cups` and `--nosocket=cups`
+X11 windowing system | Toggle | Allow the application to open in an X11 window. <br /> <br /> Most applications use X11 for historical reasons, but is considered less secure. | `--socket=x11` and `--nosocket=x11`
+Wayland windowing system | Toggle | Allow the application to open in a Wayland window. <br /> <br /> Many applications do not use Wayland as it is a newer display protocol unlike X11, and is considered more secure, but either some applications require extra steps to use it (see [environment variables](#environment) example for Firefox), or do not support Wayland at all. | `--socket=wayland` and `--nosocket=wayland`
+Fallback to X11 windowing system | Toggle | Allow the application to open in an X11 window when Wayland is not available. This overrides the X11 windowing system option when enabled. | `--socket=fallback-x11` and `--nosocket=fallback-x11`
+PulseAudio sound server | Toggle | Allow the application to play sounds when using PulseAudio. | `--socket=pulseaudio` and `--nosocket=pulseaudio`
+D-Bus session bus | Toggle | Allow the application to have access to the entire session bus. | `--socket=session-dbus` and `--nosocket=session-dbus`
+D-Bus system bus | Toggle | Allow the application to have access to the entire system bus. | `--socket=system-dbus` and `--nosocket=system-dbus`
+Secure Shell agent | Toggle | Allow the application to use SSH authentications. | `--socket=ssh-auth` and `--nosocket=ssh-auth`
+[Smart cards](https://wiki.debian.org/Smartcards) | Toggle | Allow the application to use smart cards. | `--socket=pcsc` and `--nosocket=pcsc`
+Printing system | Toggle | Allow the application to use printing systems. <br /> <br /> For example, if it's disabled for Firefox, it will no longer possible to browse the internet with this application. | `--socket=cups` and `--nosocket=cups`
 
 #### Device
 
 List of devices available in the sandbox.
```

Comment:
Can you remove these?

Reply:
Can you tell me what you are referring to?

It's hard to tell what you are reviewing. Not sure why GitHub does that.

![image](https://user-images.githubusercontent.com/50847364/98371361-0a547100-200a-11eb-9df0-fcb7689196c2.png)

Reply:
I mean, remove the permissions descriptions like `List of devices available in the sandbox.` we don't need it here.

Reply:
https://github.com/tchx84/Flatseal/pull/171/commits/50b489b7faab810aa0d9292c3d295acad92098e3